bugzilla6/Dockerfile.PL

```diff
@@ -1,20 +0,0 @@
-use Dockerfile;
-
-FROM 'centos:7';
-
-DOCKER_ENV BUGZILLA_GIT => 'git://github.com/bugzilla/bugzilla.git';
-
-RUN qq{
-    yum -y -q update &&
-    yum -y -q groupinstall  "Development Tools" &&
-    yum -y -q install
-        expat-devel
-        gd-devel
-        httpd-devel
-        mysql-devel
-        openssl
-        openssl-devel
-        perl-core
-};
-
-build_tarball();
```

Comment:
because it won't build until the XSLoader problem is fixed in the cpan ecosystem -- or untill upstream bugzilla pins its version of List::MoreUtil to a version less than the current one.